test/format.test.ts

```diff
@@ -6,6 +6,17 @@ import spawn from 'spawn-please'
 import chaiSetup from './helpers/chaiSetup'
 import stubVersions from './helpers/stubVersions'
 
+/**
+ * It is necessary to sleep on Windows to get around errors like:
+ * Error: EBUSY: resource busy or locked, rmdir 'C:\Users\alice\AppData\Local\Temp\npm-check-updates-yc1wT3'
+ */
+async function sleepOnWindows() {
+  if (process.platform === 'win32') {
+    // 100 milliseconds is arbitrarily chosen, but it seems to successfully resolve the issue.
+    await new Promise(resolve => setTimeout(resolve, 100))
```

Comment:
I would prefer to wait for a specific condition or retry rather than wait an arbitrary amount of time. 

Arbitrary sleeps add unnecessary time to test suite execution which can add up over time.